packages/ui/src/accounts/model/lockTypes.ts

```diff
@@ -57,7 +57,7 @@ const isConflictingWith = (lockTypeA: LockType): ((lockTypeB: LockType) => boole
   }
 }
 
-export const isRecoverable = (type: LockType, isActiveCandidate?: boolean): boolean => {
+export const isRecoverable = (type: LockType, isActiveCandidate?: boolean, isVoteStakeLocked?: boolean): boolean => {
```

Comment:
IMO it would be nicer to pass `isActiveCandidate` and `isVoteStakeLocked` in an object, e.g:
```suggestion
export const isRecoverable = (type: LockType, { isActiveCandidate, isVoteStakeLocked }: RecoverStakeConditions): boolean => {
```